LuckyDragon/app/src/main/java/com/example/luckydragon/Fragments/OrganizerEventFragment.java

```diff
@@ -45,6 +45,20 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         // Need a user array adapter
         entrantArrayAdapter = new EntrantArrayAdapter(event.getWaitlistUsers(), requireActivity().getApplicationContext(), this);
         waitlistListView.setAdapter(entrantArrayAdapter);
+        TextView noWaitlisteesTextView = view.findViewById(R.id.noWaitlisteesTextView);
+        if (event.getWaitlistUsers().size() == 0) {
+            noWaitlisteesTextView.setVisibility(View.VISIBLE);
+        }
+
+        // Set up InviteList list view
+        ListView invitelistListView = view.findViewById(R.id.eventInvitelistListView);
+        // Need a user array adapter
+        entrantArrayAdapter = new EntrantArrayAdapter(event.getInviteelistUsers(), requireActivity().getApplicationContext(), this);
+        invitelistListView.setAdapter(entrantArrayAdapter);
+        TextView noinviteesTextView = view.findViewById(R.id.noinviteesTextView);
+        if (event.getInviteelistUsers().size() == 0) {
+            noinviteesTextView.setVisibility(View.VISIBLE);
+        }
```

Comment:
can you try handling updating what is displayed in the OrganizerEventView if it does not result in lots of bugs?